ui/ui-config/pages/app/new/index.tsx

```diff
@@ -139,28 +154,40 @@ export default function Page(props) {
       setShowSpinner(false);
       setInstallingMessage(
         <Alert variant="warning">
-          There was a problem restarting Grouparoo. <br />
+          There was a problem restarting.
+          <br />
           <br />
-          Please restart the application via the command line.
+          Please restart the Grouparoo config app via the CLI.
         </Alert>
       );
       return;
     }
 
-    let response: Actions.PublicStatus = await client.request(
-      "get",
-      "/status/public",
-      undefined,
-      { useCache: false }
-    );
+    const retry = async () => {
+      console.log("Server down. Trying again in 1s...");
+      await sleep(1000);
+      return await waitForServer(failTime);
```

Comment:
Why not just add 1000 to failTime?

Reply:
It's because while this is happening the app is restarting so if we wait for the server it's going to fail immediately due to network errors